indexing-service/src/main/java/org/apache/druid/indexing/common/task/Task.java

```diff
@@ -184,6 +186,7 @@ default Set<ResourceAction> getInputSourceResources() throws UOE
    * MSQWorkerTask returns true from this method (because it embeds a query stack for running multi-stage queries)
    * even though it is not directly queryable via HTTP.
    */
+  @Deprecated
```

Comment:
We adjust merge pools and stuff with this method. Why is this marked deprecated. The javadocs below fail to shed light on this.

Reply:
Ah that's right, I see it in the usages. Removed the `@Deprecated` annotation.